ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/anomalylocalization/Input.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ *
+ */
+
+package org.opensearch.ml.engine.algorithms.anomalylocalization;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.opensearch.index.query.QueryBuilder;
+import org.opensearch.search.aggregations.AggregationBuilder;
+
+import lombok.Data;
+
+/**
+ * Information about aggregate, time, etc to localize.
+ */
+@Data
+public class Input {
```

Comment:
How about renaming this class as `AnomalyLocalizationInput` and implementing `org.opensearch.ml.common.parameter.Input`?

We should put input/output in `common` package.

Same comment for `org.opensearch.ml.engine.algorithms.anomalylocalization.Output`

Reply:
1. Implementing the input/output/executor interface, and all integration for ml-common framework, will be in a separate pr following all steps for integration.
2. The input and output are specific to this solution, thus in the package specifically named with the solution. I am not sure it should be separate from its own module/package and be moved to a common package as it, unlike generic Input/Output interfaces, is not common to other solutions.